controllers/stargate/stargate_controller.go

```diff
@@ -66,7 +73,6 @@ func (r *StargateReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			logger.Info("Stargate resource not found", "Stargate", req.NamespacedName)
 			return ctrl.Result{}, nil
 		} else {
-			logger.Error(err, "Failed to fetch Stargate", "Stargate", req.NamespacedName)
```

Comment:
Why this change?

Reply:
To comply with John's requirement of not adding error loggings when we already return the error, which will make it printed at a higher level.

Reply:
Ah I remember now. Thanks.